alpha_challenges/the_basics.md

```diff
@@ -153,5 +160,7 @@ end
 
 **Answer 12:**
 ```ruby
-# write your code here
+def caps(string)
```

Comment:
In Ruby there are no primitive types so this will work. However be careful calling variables things like string, num, int, etc as these will break your code in a lot of other languages! 😨

Reply:
Ah, that's true. Thank you for the reminder! I remember that in Java for example I can not name certain variables primitive types. It's good to know that there are no primitive types in Ruby!